### ISSUE TITLE
Not using translation files as fallback

### ISSUE THREAD

**Issue by kikky7** [spatie/laravel-translation-loader#37]
I just realized when I save lines in the database, i.e. with key pages.about.title it takes the translation string and outputs it on the page, but when I use key which doesn't exist in the database language_lines it just outputs that string value, i.e. @lang('pages.about.subtitle') outputs pages.about.subtitle even though there is a record
```
'about' => [ 'subtitle' => 'About subtitle' ]
```
in the lang/{language_code}/pages.php file...

**Comment by freekmurze** [comment]
Support for nested translations was added in `v1.2` of the package. Are you perhaps using an older version?

**Comment by kikky7** [comment]
I am using v1.2. The package is working; it takes translation from the database by the provided key, but it's not working with fallback, when the key does not exist in the database, but exists in the translation file. When I switch to the default Laravel Translation package it takes translations from the files.

```
"spatie/laravel-translation-loader": "^1.2",
```

**Comment by kikky7** [comment]
Using v1.2, today I found the issue: it is with nested translations (even though you said they are included with this version).

Lang file: en/pages.php
```
Translation key: 'did-you-know-1' => 'Some text...' - this example works with fallback (there is no such key in the database)

Translation key: 'did-you-know' => [ 1 => 'Some text...' ] - this example doesn't work with fallback, all I get is did-you-know.1 in my html
```

**Comment by freekmurze** [comment]
Could you PR a fix? Or a failing test for me to look at?

**Comment by uyab** [comment]
This should be fixed in 1.2.1.

**Comment by freekmurze** [comment]
Like @uyab mentioned, this should be fixed in the latest version. If this is still a problem for you after running `composer update`, reopen this issue
